app/src/main/java/com/example/itbook/ui/detailbook/DetailBookFragment.kt

```diff
@@ -159,7 +160,14 @@ class DetailBookFragment : BaseFragment(), DetailBookContract.View {
         }
     }
 
-    private fun onSimilarBooksClickListener() {}
+    private fun onSimilarBooksClickListener() {
+        book?.let {
+            val fragment = SimilarBooksFragment().apply {
+                arguments = bundleOf(Book.TITLE to it.title)
+            }
+            addFragment(fragment)
+        }
```

Comment:
Xem lại luôn conventions cách đặt tên BUNDLE_BOOK_TITLE nhé 
```suggestion
        book?.let {
            addFragment(SimilarBooksFragment.newInstance(it))
        }
```